src/zoo/owl_zoo_cmd.ml

```diff
@@ -28,47 +29,54 @@ let preprocess script =
   tmp_script
 
 
-let remove_gist gist =
-  Owl_log.debug "owl_zoo: %s removed" gist;
-  let dir = Sys.getenv "HOME" ^ "/.owl/zoo/" ^ gist in
+(* TODO: use single gid to express "removing all versions of this gist" *)
```

Comment:
I think we can agree on removing a gist id is just removing it completely from the disc. Do you agree? Because I cannot really find a convincing example for just removing one version of it. But if you have good cases to convince me, I am open to different solutions.

Reply:
The only case I can think of is removing previous versions to free disk space, but that can be more effectively achieved by `remove` all the versions and then `download` latest version. So we can agree on it.